ksqldb-cli/src/test/java/io/confluent/ksql/cli/console/ConsoleTest.java

```diff
@@ -1135,6 +1139,175 @@ public void testPrintExecuptionPlan() {
+          + "        \"logEndOffset\" : 900," + NEWLINE
+          + "        \"consumerOffset\" : 800" + NEWLINE
+          + "      }, {" + NEWLINE
+          + "        \"partition\" : 1," + NEWLINE
+          + "        \"logStartOffset\" : 50," + NEWLINE
+          + "        \"logEndOffset\" : 900," + NEWLINE
+          + "        \"consumerOffset\" : 900" + NEWLINE
+          + "      } ]" + NEWLINE
+          + "    } ]" + NEWLINE
+          + "  }," + NEWLINE
+          + "  \"warnings\" : [ ]" + NEWLINE
+          + "} ]" + NEWLINE));
+    } else {
+      assertThat(output, is("" + NEWLINE
+          + "Name                 : TestSource" + NEWLINE
+          + "Type                 : TABLE" + NEWLINE
+          + "Timestamp field      : 2000-01-01" + NEWLINE
+          + "Key format           : kafka" + NEWLINE
+          + "Value format         : avro" + NEWLINE
+          + "Kafka topic          : kadka-topic (partitions: 2, replication: 1)" + NEWLINE
+          + "Statement            : sql statement text" + NEWLINE
+          + "" + NEWLINE
+          + " Field  | Type                           " + NEWLINE
+          + "-----------------------------------------" + NEWLINE
+          + " ROWKEY | VARCHAR(STRING)  (primary key) " + NEWLINE
+          + " f_0    | VARCHAR(STRING)                " + NEWLINE
+          + "-----------------------------------------" + NEWLINE
+          + "" + NEWLINE
+          + "Queries that read from this TABLE" + NEWLINE
+          + "-----------------------------------" + NEWLINE
+          + "readId (" + AGGREGATE_STATUS +") : read query" + NEWLINE
+          + "\n"
+          + "For query topology and execution plan please run: EXPLAIN <QueryId>" + NEWLINE
+          + "" + NEWLINE
+          + "Queries that write from this TABLE" + NEWLINE
+          + "-----------------------------------" + NEWLINE
+          + "writeId (" + AGGREGATE_STATUS + ") : write query" + NEWLINE
+          + "\n"
+          + "For query topology and execution plan please run: EXPLAIN <QueryId>" + NEWLINE
+          + "" + NEWLINE
+          + "Local runtime statistics" + NEWLINE
+          + "------------------------" + NEWLINE
+          + "stats" + NEWLINE
+          + "errors" + NEWLINE
+          + "(Statistics of the local KSQL server interaction with the Kafka topic kadka-topic)" + NEWLINE
+          + NEWLINE
+          + "Consumer Group       : consumer1" + NEWLINE
+          + "Kafka topic          : kadka-topic" + NEWLINE
+          + "Max lag              : 100" + NEWLINE
+          + NEWLINE
+          + " Partition | Start Offset | End Offset | Offset | Lag " + NEWLINE
+          + "------------------------------------------------------" + NEWLINE
+          + " 0         | 100          | 900        | 800    | 100 " + NEWLINE
+          + " 1         | 50           | 900        | 900    | 0   " + NEWLINE
+          + "------------------------------------------------------" + NEWLINE));
+    }
```

Comment:
Rather than adding a whole new test, just enhance `shouldPrintTopicDescribeExtended` a non-empty list in as the last parameter of the `SourceDescriptionEntity` constructor.

Reply:
ack.